protocol/dwl-ipc-unstable-v2.xml

```diff
@@ -36,8 +36,8 @@ I would probably just submit raphi's patchset but I don't think that would be po
     </request>
 
     <request name="get_output">
-      <description summary="get a dwl_ipc_outout for a wl_output">
```

Comment:
Is `outout` some technical term from the protocol, or do you mean `output`?

Reply:
I'll prefer to don't touch this file, as it's a protocol file.